src/frontend/src/components/footer.js

```diff
@@ -72,7 +72,7 @@ const Footer = () => {
       </div>
     </div>
   </div>
-  <div className="text-center font-SourceSansPro p-6 bg-[#f0def4]">
+  <div className="text-center footerText2 font-SourceSansPro p-6 bg-[#f0def4]">
```

Comment:
The file is now renamed to `Footer.js` from `footer.js` so pull the fresh changes from master and add make sure to add these changes in `Footer.js`